### DISCUSSION TITLE
Transfer from Vinyl to mp3

### DISCUSSION THREAD

**Discussion by afoster1955** [enzo1982/freac#525] [General Discussion]
I just recently downloaded fre:ac and am wondering if it can be used to transfer music from vinyl record to laptop.  In looking at the directions, it doesn't appear that it is possible but thought I would ask.

**Comment by Lee-Carre** [comment]
> […] transfer music from vinyl record to laptop. […] it doesn't appear that it is possible […]

Not in the same way that can be done with/from CDs, no. They're fundamentally different source formats, though. For example, matching CDDB metadata (track names and such) involves using non-audio (meta)data (disc ID, for example) from the CD. Vinyl (as far as I know) has only the audio (well, no machine-readable data, at least; identifying info is printed on the label, and would have to be read/entered manually; but see below about MusicBrainz Picard for tagging).

However, if you connect your turntable output via suitable equipment (pre-amp and such; that's a different conversation) to an audio input on your host, you could then record that using something like [Audacity](//www.audacityteam.org/).

Have Audacity save using an uncompressed format.

You could then use Fre:AC to transcode to whatever format you like (FLAC, Opus, or MP3 if you must). Fre:AC is apt to batch processing many files, which is likely what you want.

For tagging, I suggest [MusicBrainz Picard](//picard.musicbrainz.org/). Picard includes a feature to identify music based on acoustic signature (the audio waveform), in order to find matches in the database (which includes vinyl editions of albums; else you could contribute metadata for albums you have which are missing from the database).

I would suggest/recommend using some shell-scripting to somewhat automate the process, if you have many vinyl discs you wish to digitise.

**Comment by afoster1955** [comment]
Thanks for getting back to me.  What I am looking for (and evidently not doable) is a piece of software that would manage the conversion of vinyl sound to mp3 without using Audacity. Something along the lines of EZ vinyl/tape converter software.  I was hoping that Fre:ac would to that but as I said in my original post, I didn't think it was possible. I have used Fre:ac to rip music from CD's however, and that works great.
